charter/2023-09-01.md

```diff
@@ -163,11 +163,10 @@ An affiliation MAY submit one ballot that ranks candidates in their preferred or
 
 #### Choosing a Chair
 
-* At any given time there may be up to three co-chairs, each holding one seat. Each seat defines a 2 year cycle of service.
-* In the first election after ratification of this charter, all seats will be up for election. Thereafter, in each year, a single election will be held to fill any vacant seats.
-* In the case of interim vacancy, the remaining chairs may appoint a co-chair for each open seat, hold an election for the same, or wait until the next election, at their discretion. If the chairs do not take any action, the seat will automatically be up for election in any cycle. Any such interim appointments or elections shall hold the seat until the end of its natural cycle.
+* At any given time there may be up to three co-chairs, each holding one seat for a 2 year cycle, or shorter in case they step down or are removed.
+* In the first election after ratification of this charter, all seats will be up for election. Thereafter, elections will be held whenever there are vacant seats.
 * Reelection is restricted to two consecutive terms, with the possibility of being reelected after sitting out one election cycle.
```

Comment:
It's unclear whether these "two consecutive terms" are two *full* terms, or two elections, even if the first is to fill a mid-term vacancy, no matter how few months remain in that term. Note that, for instance, the US president is restricted to two elections to full terms but these may be in addition to a full-term-minus-one-day, if that results from succession to a vacancy by resignation, death, impeachment, etc.